buildsystem/cythonize.py

```diff
@@ -11,6 +11,49 @@
 import sys
 
 
+class LineFilter:
+    """ Proxy for a stream (default stdout) to filter out whole unwanted lines """
+    # https://stackoverflow.com/questions/34904946/how-to-filter-stdout-in-python-logging
+    def __init__(self, stream=None, filtered=None):
+        self.stream = stream or sys.stdout
+        self.filtered = filtered or []
+        self.buf = ""
+
+    def __getattr__(self, attr_name):
+        return getattr(self.stream, attr_name)
+
+    def write(self, data):
+        """
+        Writes to output stream, buffered linewise,
+        omitting lines given in to constructor in filter
+        """
+        self.buf += data
+        lines = self.buf.split('\n')
+        for line in lines[:-1]:
+            if line not in self.filtered:
+                self.stream.write(line + '\n')
+        self.buf = lines[-1]
+
+    def flush(self):
+        """ Flushes output stream """
+        self.stream.flush()
```

Comment:
i think we can [remove this method](https://docs.python.org/3/reference/datamodel.html#object.__getattr__)